library/src/main/java/com/novoda/downloadmanager/Migrator.java

```diff
@@ -5,7 +5,7 @@
     void migrate();
 
     interface Callback {
-        void onMigrationComplete();
```

Comment:
`onMigrationComplete` was exposed originally so we could delete the database using the `ContextWrapper`, found out we can do it by just deleting the `file` which doesn't require access to `Context` 💥